pkgs/development/python-modules/anyqt/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "AnyQt";
```

Comment:
In general, pname is lowercase. Only exceptions afaicr are SDL and extensions.a

Reply:
Done, also did a `git diff master | grep pname` to check if there is anything else